projects/core/src/lib/image-resizer.ts

```diff
@@ -104,6 +104,6 @@ function canvasToFile(canvas: HTMLCanvasElement, name: string, type: string, qua
       }, type, quality);
     });
   } else {
-    return of(blobToFile(canvas.msToBlob(), type, name));
+    return of(blobToFile((<any>canvas).msToBlob(), type, name));
```

Comment:
This caused the build to fail in Angular 7, while it seems to work fine with Angular 6.